src/realm/util/util_websocket.cpp

```diff
@@ -1,4 +1,4 @@
-#include <realm/util/ez_websocket.hpp>
+#include <realm/util/util_websocket.hpp>
```

Comment:
util/util_websocket.hpp seems redundant and confusing to me. I know we've gone through some names already, but how about using a "client" prefix on this file and as a substitute for "EZ"? Looks like the other implementation are only kept around to build the legacy test server.